run.py

```diff
@@ -486,6 +488,7 @@ def run(args):
     enable_eus = args.get("--enable-eus")
     skip_enabling_rhel_rpms = args.get("--skip-enabling-rhel-rpms")
     skip_sos_report = args.get("--skip-sos-report")
+    skip_subcommands = args.get("--skip-subcommands")
```

Comment:
we already have lot of parameters to run.py, kindly avoid adding unnecessary parameters.

Reply:
@psathyan ^^^^

Reply:
Noted @vamahaja we will run this only during rc and we will remove this parameter